openid-sharedsignals-framework-1_0.md

```diff
@@ -537,42 +548,46 @@ critical_subject_members
 
 > OPTIONAL. An array of member names in a Complex Subject which, if present in
   a Subject Member in an event, MUST be interpreted by a Receiver.
-  
-supported_scopes
 
-> OPTIONAL. A list of OAuth {{RFC6749}} scope names that the Transmitter supports for specific endpoints. The value of this field is a JSON object that has the endpoint names as keys, and arrays of scope name strings they support as their values. OAuth tokens obtained using any of the scopes defined here MUST be accepted by the specified endpoint. Any key that is not defined as an endpoint in the Transmitter Configuration Metadata MUST be ignored. If the supported_scopes member is present in the metadata and an endpoint is not listed as a key, then that endpoint MUST not require OAuth for authorization.
+authorization_schemes
 
-authorization_servers
+> OPTIONAL. A multi-valued complex type that specifies supported
+  authorization scheme properties defined in {{authorization-scheme}}.  To enable seamless discovery of
+  configurations, the service provider SHOULD, with the appropriate
+  security considerations, make the authorization_schemes attribute
+  publicly accessible without prior authentication.
 
-> OPTIONAL. An array of supported authorization servers and the scopes they support. Each element of the array is a Authorization Server Descriptor JSON object defined in the section {{authz-server-descriptor}} below. If the `supported_scopes` member is present in the metadata, then the `authorization_servers` MUST also be present, and it MUST provide a server location for every supported scope.
 
 TODO: consider adding a IANA Registry for metadata, similar to Section 7.1.1 of
 {{RFC8414}}. This would allow other specs to add to the metadata.
 
-### Authorization Server Descriptor {#authz-server-descriptor}
-An Authorization Server Descriptor is a JSON object that has two keys:
+### Authorization scheme {#authorization-scheme}
+SSF is an HTTP based signals sharing framework. It is agnostic to the authentication and authorization schems used to secure stream configuration APIs. It does not provide any SSF specific authentication and authorization schemes but relies on the cooperating parties mutual security considerations. Authorization scheme section of the metadata, providers disovery informaton related to the transmitter's stream management APIs. 
 
-scopes
+type
 
-> REQUIRED. An array of scope names supported by the authorization server
+> The authorization scheme.  This specification defines the values "oauth",
```

Comment:
Should we just put the URN / URI of the spec here (like we do for push and poll delivery)? That way we do not need additional fields.

Reply:
OAUTH2 metadata & discovery should be optional in the core spec.  Everything can be agreed out-of-band for some implementations.   What we will need to layer ontop of the core spec is an OAUTH2 profile for SSF that defines mandatory OAUTH2 capabilities for interoperability.   This is where we can be optioned about what is mandatory.

Reply:
1. @atultulshi I thought about it but may not fit in custom/proprietary schemes. Let me know any suggestions around this. Maybe custom schemes could be documented and populated used `documentation_uri`
2. @mcguinness please note that `authorization_schemes` itself is marked `OPTIONAL`, this would decouple any authZ schemes from SSF core spec

Reply:
Sorry, thought `Should we just put the URN / URI of the spec here` was referring to the discovery spec vs the OAUTH2 spec.  Misread.

Reply:
Updated with the `spec_urn` @atultulshi